examples/tutorials/Scanpy.ipynb

```diff
@@ -829,32 +837,30 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In some ocassions, you might still observe disconnected clusters and similar connectivity violations. They can usually be remedied by running:"
```

Comment:
Let's not delete this

Reply:
I have actually shifted this code block ahead, this is due to the usage of paga() which requires a categorical key to be mentioned. Hence, I have shifted this after we have created the leiden clustering variable.